src/pages/_app.tsx

```diff
@@ -1,17 +1,34 @@
+import React, { useEffect, useState } from 'react'
+import type { UseState } from '@okp4/ui'
 import type { AppProps } from 'next/app'
 import dynamic from 'next/dynamic'
 import { Layout } from '../components'
+import type { Config } from './api/config'
 import './styles.scss'
 
 const ThemeProvider = dynamic(async () => (await import('@okp4/ui')).ThemeProvider, {
   ssr: false
 })
 
+const fetchConfig = async (): Promise<Config> => {
+  const response = await fetch('/api/config')
+  const config: Config = await response.json()
```

Comment:
Here you can directly return the value

```suggestion
  return await response.json()
```

Reply:
@RaphaelEscrig It's better to keep `const config` because it allows us to type what the function returns, otherwise it is typed as `any`.